ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Deployment/DeploymentPageResources.tsx

```diff
@@ -1,11 +1,120 @@
+import ImageResourceTable, { ImageResources, imageResourcesFragment } from './ImageResourceTable';
+
+export type DeploymentPageResourcesProps = {
+    deploymentId: string;
+};
+
+const deploymentResourcesQuery = gql`
+    ${imageResourcesFragment}
+    query getDeploymentResources($id: ID!, $query: String, $pagination: Pagination) {
+        deployment(id: $id) {
+            id
+            ...ImageResources
+        }
+    }
+`;
+
+function DeploymentPageResources({ deploymentId }: DeploymentPageResourcesProps) {
+    const { page, perPage, setPage, setPerPage } = useURLPagination(20);
+    const { sortOption, getSortParams } = useURLSort({
+        sortFields: defaultImageSortFields,
+        defaultSortOption: imagesDefaultSort,
+        onSort: () => setPage(1),
+    });
+
+    const imageTableToggle = useSelectToggle(true);
+
+    const { data, previousData, loading, error } = useQuery<
+        { deployment: ImageResources | null },
+        { id: string; query: string; pagination: PaginationParam }
+    >(deploymentResourcesQuery, {
+        variables: {
+            id: deploymentId,
+            query: '',
+            pagination: {
+                offset: (page - 1) * perPage,
+                limit: perPage,
+                sortOption,
+            },
+        },
+    });
+
+    const deploymentResourcesData = data?.deployment ?? previousData?.deployment;
+    const imageCount = deploymentResourcesData?.imageCount ?? 0;
 
-function DeploymentPageResources() {
     return (
         <>
-            <div>DeploymentPageResources</div>
+            <PageSection component="div" variant="light" className="pf-u-py-md pf-u-px-xl">
+                <Text>Navigate to resources associated with this deployment</Text>
+            </PageSection>
+            <Divider component="div" />
+            <PageSection
+                className="pf-u-display-flex pf-u-flex-direction-column pf-u-flex-grow-1"
+                component="div"
+            >
+                {error && (
+                    <TableErrorComponent
+                        error={error}
+                        message="Adjust your filters and try again"
+                    />
+                )}
+                {loading && !deploymentResourcesData && (
+                    <Bullseye>
+                        <Spinner isSVG />
+                    </Bullseye>
+                )}
+                {deploymentResourcesData && (
+                    <ExpandableSection
+                        toggleText={`Images (${imageCount})`}
+                        onToggle={imageTableToggle.onToggle}
+                        isExpanded={imageTableToggle.isOpen}
+                        style={
+                            {
+                                '--pf-c-expandable-section__content--MarginTop':
+                                    'var(--pf-global--spacer--xs)',
+                            } as CSSProperties
```

Comment:
oh nifty 👀  i feel like we usually just override in the css directly instead of the style, but this works too!